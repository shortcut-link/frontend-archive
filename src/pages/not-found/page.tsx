import React from 'react';

export const NotFoundPage: React.FunctionComponent = () => (
  <div>Not found page.</div>
);
