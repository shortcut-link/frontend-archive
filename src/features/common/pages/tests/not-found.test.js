import React from 'react';
import renderer from 'react-test-renderer';

import { NotFoundPage } from '../not-found';

it('must render NotFoundPage', () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
