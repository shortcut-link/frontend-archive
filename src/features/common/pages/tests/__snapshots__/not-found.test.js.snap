// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the page "not found page" 1`] = `
<div>
  Not found page.
</div>
`;
